avm/utilities/e2e-template-assets/templates/azure-stack-hci/modules/microsoftGraphResources/bicepconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "experimentalFeaturesEnabled": {
+        "extensibility": true
+    }
```

Comment:
I'm not sure why this is here, but @jtracey93, I guess we don't allow experimental features per se, right?

Reply:
I removed the MS Graph resources